package.json

```diff
@@ -34,6 +34,7 @@
     "newrelic": "^9.0.0",
     "passport": "^0.6.0",
     "passport-github2": "^0.1.12",
+    "rate-limiter-flexible": "^2.4.1",
```

Comment:
try excluding the caret symbol in package json

Reply:
hey @nitiiink what's the need of excluding, do you want to manage updates manually?

Reply:
hey @shmbajaj yes, usually there might be vulnerabilities associated with the codebase if we keep automatic updates on,hence suggested this.

Reply:
Thanks @nitiiink, I will implement in the subsequent feature PR.